### DISCUSSION TITLE
Mongodb docker couldn’t start up

### DISCUSSION THREAD

**Discussion by qtqtho** [docker-library/mongo#594] [General]
Hello, I’m new to mongodb docker so when I try to start up the mongodb docker container using docker-compose up, the container exit with code 14. Couldn’t find a solution to this please help.

**Comment by wglambert** [comment]
Can you give all the commands you ran and any relevant files and `docker logs` for reproducing the issue

You could also try asking over at the [Docker Community Forums](https://forums.docker.com/), [Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker). Since these repos aren't really a user-help forum
